operator/builtin/transformer/move/move.go

```diff
@@ -0,0 +1,73 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package move
+
+import (
+	"context"
+
+	"github.com/open-telemetry/opentelemetry-log-collection/entry"
+	"github.com/open-telemetry/opentelemetry-log-collection/operator"
+	"github.com/open-telemetry/opentelemetry-log-collection/operator/helper"
+)
+
+func init() {
+	operator.Register("move", func() operator.Builder { return NewMoveOperatorConfig("") })
+}
+
+// NewMoveOperatorConfig creates a new restructure operator config with default values
+func NewMoveOperatorConfig(operatorID string) *MoveOperatorConfig {
+	return &MoveOperatorConfig{
+		TransformerConfig: helper.NewTransformerConfig(operatorID, "move"),
+	}
+}
+
+// MoveOperatorConfig is the configuration of a restructure operator
+type MoveOperatorConfig struct {
+	helper.TransformerConfig `mapstructure:",squash" yaml:",inline"`
+	From                     entry.Field `mapstructure:"from" yaml:"from"`
+	To                       entry.Field `mapstructure:"to" yaml:"to"`
+}
+
```

Comment:
Please add comments to exported methods and structs: https://blog.golang.org/godoc